packages/openid4vc/src/openid4vc-issuer/repository/OpenId4VcIssuerRecord.ts

```diff
@@ -0,0 +1,60 @@
+import type { OpenId4VciCredentialSupportedWithId, OpenId4VciIssuerMetadataDisplay } from '../../shared'
+import type { RecordTags, TagsBase } from '@aries-framework/core'
+
+import { BaseRecord, utils } from '@aries-framework/core'
+
+export type OpenId4VcIssuerRecordTags = RecordTags<OpenId4VcIssuerRecord>
+
+export type DefaultOpenId4VcIssuerRecordTags = {
+  issuerId: string
+}
+
+export interface OpenId4VcIssuerRecordProps {
+  id?: string
+  createdAt?: Date
+  tags?: TagsBase
+
+  issuerId: string
+
+  /**
+   * The fingerprint (multibase encoded) of the public key used to sign access tokens for
+   * this issuer.
+   */
+  accessTokenPublicKeyFingerprint: string
+
+  credentialsSupported: OpenId4VciCredentialSupportedWithId[]
+  display?: OpenId4VciIssuerMetadataDisplay[]
+}
+
+export class OpenId4VcIssuerRecord extends BaseRecord<DefaultOpenId4VcIssuerRecordTags> {
```

Comment:
I think this could benefit from a comment. We never really had an `issuer` record for DIDComm and I am a bit confused whether an issuer should only have one record or multiple for different servers (if that even makes sense)?

Reply:
Will add. The generic thing is that for OID4VC you need to expos metadata files. Each issuer needs to host this metadata. This is not the case for DIDComm where we can just have one `/didcomm` endpoint. So we create a record per openid issuer/verifier that you want, and each tenant can create multiple issuers/verifiers